mozilla_aws_cli/login.py

```diff
@@ -414,13 +424,9 @@ def print_output(self):
 
             if self.output == "envvar":
                 output_map.update(
-                    {ENV_VARIABLE_NAME_MAP[x]: self.credentials[x]
-                     for x in self.credentials
-                     if x in ENV_VARIABLE_NAME_MAP})
-                output_map.update({
-                    "AWS_PROFILE": None,
-                    "AWS_SHARED_CREDENTIALS_FILE": None,
-                    "MAWS_PROMPT": self.display_name})
+                    {var: self.credentials.get(key)
+                     for key, var in ENV_VARIABLE_NAME_MAP.item()})
```

Comment:
```python
Encountered error : Unable to contact AWS : Traceback (most recent call last):
File "/usr/local/lib/python3.9/dist-packages/mozilla_aws_cli-1.2.4-py3.9.egg/mozilla_aws_cli/login.py", line 363, in exchange_token_for_credentials
self.print_output()
File "/usr/local/lib/python3.9/dist-packages/mozilla_aws_cli-1.2.4-py3.9.egg/mozilla_aws_cli/login.py", line 428, in print_output
for key, var in ENV_VARIABLE_NAME_MAP.item()})
AttributeError: 'dict' object has no attribute 'item'
```

Reply:
Sorry about that! Should be `.items()`. Fixed now (togerther with a few other clean-ups). This time I at least did a basic test that it's still working.